src/models/courseItem.ts

```diff
@@ -11,7 +11,7 @@ export const courseItemSchema = new dynamoose.Schema(
 		name: String,
 		weight: Number,
 		grade: [Number, fraction],
-		dueDate: Date,
+		date: [String, Date],
```

Comment:
Just wondering why this change was necessary. If when we load the data from the database it is returned as a string initially we can modify it to be a date immediately if needed, though it will end being turned into a string for the request I think.

Reply:
Honestly I just changed this in order to make sure the calls worked in the first place - this should definitely be changed back, it was just a way to deal with typing issues without actually solving the problem in the first place (in other words, I was too lazy to actually convert the date from the add item modal into a date). Should I try and solve the issue on this ticket? Or would it be best in the scope of a smaller ticket?